pipelines/datasets/au_ato_taxation_statistics/constants.py

```
@@ -0,0 +1,150 @@
+
+    # Resource selectors, anchored on BOTH the table number and its
+    # descriptive slug. Anchoring on the number alone is unsafe: the ATO
+    # reused "gst4" for petroleum resource rent tax before 2014-15 and for
+    # the by-industry table afterwards.
+    TABLE_SELECTORS = {
+        "individuals_income_state": r"individual04(sex|gender)",
+        "individuals_industry": r"individual05(sex|gender)",
+        "individuals_postcode": r"individual06taxablestatus",
+        "company_industry": r"company04industry",
+        "gst_industry": r"gst04byindustry",
+    }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)